site/docs/master/restore-reference.md

```diff
@@ -9,6 +9,44 @@ velero restore create RESTORE_NAME \
   --from-backup BACKUP_NAME \
   --namespace-mappings old-ns-1:new-ns-1,old-ns-2:new-ns-2
 ```
+## What happens when user removes restore Objects. 
+A **restore** object represents a restore operation.There are two types of deletion for restore objects:
+### 1. Deleting with **`velero restore delete`**
+This command will delete the custom resource representing it, along with its individual log and results files.But, it will not delete any objects that were created by it from your cluster.
+### 2. Deleting with **`kubectl -n velero delete restore`**
+This command will delete the custom resource representing the restore, but will not delete log/results files from object storage, or any objects that were created by the restore in your cluster.
```

Comment:
instead of "...created by the restore...", let's say "created during the restore"